Client/Frontend/Browser/Playlist/BrowserViewController+Playlist.swift

```diff
@@ -7,81 +7,145 @@ import Foundation
 import Data
 import Shared
 import BraveShared
+import BraveUI
 
 private let log = Logger.browserLogger
 
 extension BrowserViewController: PlaylistHelperDelegate {
-    func showPlaylistAlert(_ alertController: UIAlertController) {
-        self.present(alertController, animated: true)
-    }
     
-    func showPlaylistToast(info: PlaylistInfo, itemState: PlaylistItemAddedState) {
-        guard Preferences.Playlist.showToastForAdd.value,
-              let selectedTab = tabManager.selectedTab,
-              selectedTab.url?.isPlaylistSupportedSiteURL == true else {
-            return
+    func updatePlaylistURLBar(tab: Tab?, state: PlaylistItemAddedState, item: PlaylistInfo?) {
```

Comment:
looks like we can only pass Tab's url here instead of entire object

Reply:
No because I need to check if the tab is the selected tab. If it isn't, I need to do different logic so that the URL bar isn't updated for the wrong tab. This has been addressed in the last commit :)